camkes/templates/component.common.c

```diff
@@ -74,6 +74,14 @@ const char *get_instance_name(void) {
     return name;
 }
 
+int get_instance_core(void) {
+/*- if 'affinity' in configuration[me.name].keys() -*/
```

Comment:
The `.keys()` here is redundant.

Reply:
nit: but maybe more readable     return /*? configuration[me.name].get('affinity', 0) ?*/;

Reply:
Sounds good.